morango/models.py

```diff
@@ -100,10 +96,33 @@ class AbstractStoreModel(models.Model):
     last_saved_instance = models.UUIDField()
     last_saved_counter = models.IntegerField()
     last_saved_counter_per_instance = models.TextField(default="{}")  # RMC
+    model_name = models.CharField(max_length=40)
+
+    class Meta:
+        abstract = True
+
+
+class AbstractDatabaseMaxCounter(models.Model):
+
+    instance_id = models.UUIDField()
+    max_counter = models.IntegerField()
+    transfer_session_id = UUIDField()
+    kind = models.BooleanField()
 
     class Meta:
         abstract = True
 
+    @classmethod
+    def filter_max_counters(cls, query):
+        query = json.loads(query)
```

Comment:
Better to have it be a dict already -- on the side that's creating the filter specification/query, it shouldn't have to first convert to a string before using this function.